### ISSUE TITLE
Fantastic work, can you tell me how you analyzed it

### ISSUE THREAD

**Issue by qianmo-xw** [R0rt1z2/oplus-unlock#7]
I have an OPPO Qualcomm machine, I don't know what the reason is, it may be that the key for fastboot verification is lost, and now I can't enter fastboot, but BL is still unlocked, so can I use the same method?

**Comment by R0rt1z2** [comment]
Qualcomm devices don't use LK as their secondary bootloader. Even if you were to successfully modify whatever the current bootloader is (I believe abl), you'd still face the challenge of flashing the patched image onto the device.

**Comment by qianmo-xw** [comment]
> Qualcomm devices don't use LK as their secondary bootloader. Even if you were to successfully modify whatever the current bootloader is (I believe abl), you'd still face the challenge of flashing the patched image onto the device.

Yes, you are right, but I have a relatively new machine. I can use fastbootd mode to flash the partition. The current problem is how to modify abl. I have no clue. Are you interested? Can I send you the abl.elf or can you tell me how to find the check bit to turn off OPPO?

**Comment by R0rt1z2** [comment]
> > Qualcomm devices don't use LK as their secondary bootloader. Even if you were to successfully modify whatever the current bootloader is (I believe abl), you'd still face the challenge of flashing the patched image onto the device.
> 
> Yes, you are right, but I have a relatively new machine. I can use fastbootd mode to flash the partition. The current problem is how to modify abl. I have no clue. Are you interested? Can I send you the abl.elf or can you tell me how to find the check bit to turn off OPPO?

Pretty sure SBC (secure boot) will brick your phone, but if you still want to try it, send me the image and I'll try to analyze it.

**Comment by qianmo-xw** [comment]
`

> > > Qualcomm devices don't use LK as their secondary bootloader. Even if you were to successfully modify whatever the current bootloader is (I believe abl), you'd still face the challenge of flashing the patched image onto the device.
> > 
> > 
> > Yes, you are right, but I have a relatively new machine. I can use fastbootd mode to flash the partition. The current problem is how to modify abl. I have no clue. Are you interested? Can I send you the abl.elf or can you tell me how to find the check bit to turn off OPPO?
> 
> Pretty sure SBC (secure boot) will brick your phone, but if you still want to try it, send me the image and I'll try to analyze it.

[abl.zip](https://github.com/R0rt1z2/oplus-unlock/files/12708316/abl.zip)
This file. Thank you.

**Comment by R0rt1z2** [comment]
Couldn't figure anything out, sorry.
